docs/develop/understanding-the-javascript-api-for-office.md

```diff
@@ -48,7 +48,7 @@ If you fail to assign an event handler, your add-in may raise an error when it s
 For more detail about the sequence of events when an add-in is initialized, see [Loading the DOM and runtime environment](../../docs/develop/loading-the-dom-and-runtime-environment.md).
 
 #### Initialization Reason
-For Take Pane and Content add-ins, Office.initialize provides an additional _reason_ parameter. This parameter can be used to determine how an add-in was added to the current document. You can use this to provide different logic for when an add-in is first inserted versus when it already existed within the document. 
+For Task Pane and Content add-ins, Office.initialize provides an additional _reason_ parameter. This parameter can be used to determine how an add-in was added to the current document. You can use this to provide different logic for when an add-in is first inserted versus when it already existed within the document. 
```

Comment:
FYI, we don't capitalize "task pane" and "content" when we're talking about the types of add-ins.